### ISSUE TITLE
External map links

### ISSUE THREAD

**Issue by fisharebest** [fisharebest/webtrees#2442]
When an event has latitude/longitude co-ordinates, webtrees shows 3 links to external mapping providers; Google, Bing and OpenStreetMap.

We should just show one link - and it should be configurable.

**Comment by wkitty42** [comment]
On 5/17/19 4:35 AM, Greg Roach wrote:
> When an event has latitude/longitude co-ordinates, webtrees shows 3 links to 
> external mapping providers; Google, Bing and OpenStreetMap.
> 
> We should just show one link - and it should be configurable.


i'm not so sure about that... some folks (users) prefer to use bing while others 
prefer to use google... having all three links available allows the users to 
choose their poison instead of the operator forcing it on them... if they don't 
like the choice the operator chose, they may very well not even use the links at 
all...

TLDR; choice is good... allow the users to select whichever link they prefer to 
use... if it is configurable, it should be per user configurable with maybe a 
default chosen by the operator...
